outputs.tf

```diff
@@ -87,10 +78,13 @@ output "permissions_boundary_policy_arn" {
   description = "ARN of the policy boundary which is required to be included on any roles created by the Redpanda agent"
 }
 
-
 output "private_subnet_arns" {
-  description = "List of ARNs of the private subnets"
+  description = "Private subnet ARNs created"
```

Comment:
nit: These are not necessarily created by this module. If the variable `private_subnet_ids` was provided no new subnets are created and the ARNs corresponding to those IDs are what is returned here.